scripts/verify-tests.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+set -e
+
+echo "running tests..."
+gotestcover -race -covermode=count -coverprofile=cover.out $(go list ./... | grep -v /vendor/)
+
+echo ""
+go tool cover -func cover.out
```

Comment:
If its okay with you I would rather keep it separate in the aim of keeping the makefile as succinct as possible? I'll merge in the meantime and discuss with you tomorrow.